### PR TITLE
[TSK-172] create buildSrc module and move versions

### PR DESCRIPTION
**Pull request by Hoocki**

### CODE REVIEW

File: libs/build.gradle.kts
Comment (adsukharev) [inline libs/build.gradle.kts:18]: use direct import

**Review by adsukharev — APPROVED** [review]
change target branch to your feature branch.
Don't merge, until Kostya will merge his task

**Review by kostya4201337 — APPROVED** [review]

**Pull request closed without merging**
